modules/ROOT/partials/toolbar-button-ids/uploadcare-toolbar-buttons.adoc

```diff
@@ -0,0 +1,5 @@
+[cols="1,3",options="header"]
+|===
+|Identifier |Description
+| `+uploadcare+` | Inserts a `<p>` tag with the `+contenteditable="false"+` attribute, containing a placeholder image. This placeholder supports image uploads via drag-and-drop, file selection, or by providing a URL.
```

Comment:
I don't thing we need to go deep on the details here.

```suggestion
| `+uploadcare+` | Inserts a image placeholder element. This placeholder supports image uploads via drag-and-drop, file selection, or by providing a URL.
```

Reply:
do they need to apply the same change to: modules/ROOT/partials/menu-item-ids/uploadcare-menu-items.adoc ?

Reply:
Updated in last commit for both `menu-items` and `toolbar-buttons`